examples/gatsby/gatsby-node.js

```diff
@@ -0,0 +1,16 @@
+const webpack = require('webpack');
```

Comment:
I tried doing this with a ts file but couldn't get it to work so have fallen back to using a js file here which does seem to work